data/live-build-config/package-lists/vyos-utils.list.chroot

```diff
@@ -2,3 +2,4 @@ systemd-sysv
 systemd-bootchart
 ncurses-term
 kitty-terminfo
+binutils
```

Comment:
Is this really required within the resulting image? 

Can't this be added in the debootstrap process?
https://github.com/vyos/vyos-build/blob/current/scripts/build-vyos-image#L436

Reply:
Or even perhaps just in the Dockerfile?

Reply:
It is required since the script `data/live-build-config/hooks/live/99-strip-symbols.chroot` is runned within the chroot (only whats inside that chroot will be available as a binary) by Debian live-build. Including it in the Dockerfile wont help (already tried that).

The only hooks available are for the chroot and the binary process. If runned as binary hook its too late because then the filesystem.squashfs (which contains the chroot) is already created.

You can see the order between chroot and binary process here:

https://salsa.debian.org/live-team/live-build/-/blob/master/scripts/build/build

And then within the binary process itself you can see that binary_rootfs (who creates the filesystem.squashfs) is runned much sooner than binary_hooks:

https://salsa.debian.org/live-team/live-build/-/blob/master/scripts/build/binary

So the trick is to include binutils package so it becomes part of the chroot and once the script is complete (which is why its started last out of the chroot scripts) it will remove the binutils package (before the filesystem.squashfs is created).

Reply:
> it will remove the binutils package (before the filesystem.squashfs is created).

+1